src/index.html

```diff
@@ -1,22 +1,29 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    />
-    <meta
-      http-equiv="X-UA-Compatible"
-      content="ie=edge"
-    />
-    <title>Moyo header</title>
-    <link
-      rel="stylesheet"
-      href="./style.css"
-    />
-  </head>
-  <body>
-    <h1>Moyo header</h1>
-  </body>
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Header</title>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <header>
+    <a href="#" class="logo">
+      <img src="images/logo.png" alt="logo">
+    </a>
+    <nav>
+      <ul>
+        <li><a href="#" class="is-active" data-qa="hover">APPLE</a></li>
+        <li><a href="#">SAMSUNG</a></li>
+        <li><a href="#">SMARTPHOMES</a></li>
+        <li><a href="#" class="special">LAPTOPS e COMPUTERS</a></li>
```

Comment:
There are spaces in the `href` attribute of the `a` tag. Anchor links should start with the `#` symbol, and spaces are not allowed.